README.md

```diff
@@ -25,10 +25,12 @@ The generated code depends on the com.shopify.graphql.support java
 package. This can be added to a gradle project by adding the following
 jCenter dependancy to you `build.gradle` file:
 
-    compile 'com.shopify.graphql.support:support:0.1.0'
+    compile 'com.shopify.graphql.support:support:0.2.0'
```

Comment:
Why did this get bumped before 0.2.0 was released?  https://bintray.com/shopify/shopify-java/graphql-support shows the latest version is 0.1.1 on bintray, so it looks like this is causing more confusion (e.g. https://github.com/Shopify/graphql_java_gen/issues/53)